google-cloud-clients/google-cloud-bigquery/src/test/java/com/google/cloud/bigquery/BigtableOptionsTest.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigquery;
+
+import static junit.framework.TestCase.assertEquals;
+import static junit.framework.TestCase.assertTrue;
+
+import com.google.common.collect.ImmutableList;
+import org.junit.Test;
+
+public class BigtableOptionsTest {
+
+
+  private static final BigtableColumn COL1 = BigtableColumn.newBuilder()
+      .setQualifierEncoded("aaa")
+      .setFieldName("field1")
+      .setOnlyReadLatest(true)
+      .setEncoding("BINARY")
+      .setType("BYTES")
+      .build();
+  private static final BigtableColumn COL2 = BigtableColumn.newBuilder()
+      .setQualifierEncoded("bbb")
+      .setFieldName("field2")
+      .setOnlyReadLatest(true)
+      .setEncoding("TEXT")
+      .setType("STRING")
+      .build();
+  private static final BigtableColumnFamily TESTFAMILY = BigtableColumnFamily.newBuilder()
+      .setFamilyID("fooFamily")
+      .setEncoding("TEXT")
+      .setOnlyReadLatest(true)
+      .setType("INTEGER")
+      .setColumns(ImmutableList.of(COL1, COL2))
+      .build();
+  private static final BigtableOptions OPTIONS = BigtableOptions.newBuilder()
+      .setIgnoreUnspecifiedColumnFamilies(true)
+      .setReadRowkeyAsString(true)
+      .setColumnFamilies(ImmutableList.of(TESTFAMILY))
+      .build();
+
+
+  @Test
+  public void testConstructors() {
+
+    // column
+    assertEquals(COL1.getQualifierEncoded(), "aaa");
```

Comment:
We have been trying to move to `assertThat(COL1.getQualifierEncoded()).isEqualTo("aaa")` could you make the change here?